src/constants/constants.ts

```diff
@@ -142,4 +153,36 @@ export const POLISH_BANKS = [
     notDoableNumber: 'not doable number',
     individualPlan: 'individual plan %',
     teamPlanContribution: 'team plan contribution',
-  };
\ No newline at end of file
+  };
+
+  export const analyticsGetColor = (el: { label: string; value: number | string }, label1:string, label2:string, plan?:string) => {
+    const HALF = 0.5;
+    const HUNDRED_PRECENT = 100
+    const value = parseDynamic(el.value);
+    const choosedPlan = plan==="team"
+? TEAM_PLAN
+: AGENT_PLAN
+    
+    if (el.label === label1) {
+        if (value >= HUNDRED_PRECENT) {
+          return 'analyticsGreen';
+        } else if (value >= HUNDRED_PRECENT * HALF) {
+          return 'analyticsOrange';
+        } else {
+          return 'analyticsRed';
+        }
+      }
+      
+      if (el.label === label2) {
+        if (value >= choosedPlan) {
+          return 'analyticsGreen';
+        } else if (value >= choosedPlan * HALF) {
+          return 'analyticsOrange';
+        } else {
+          return 'analyticsRed';
+        }
+      }
```

Comment:
```suggestion
   function getColor(label, value, threshold) {
  if (value >= threshold) {
    return 'analyticsGreen';
  } else if (value >= threshold * 0.5) {
    return 'analyticsOrange';
  } else {
    return 'analyticsRed';
  }
}

if (el.label === label1) {
  return getColor(label1, value, HUNDRED_PRECENT);
}

if (el.label === label2) {
  return getColor(label2, value, choosedPlan);
}
```

chodzi mi o to żeby to uprościć jakoś

Reply:
może nawet zamienić miejscami najpierw value a potem label?